examples/SHT3X/main/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+idf_component_register(
+    SRCS "main.cpp"
+    INCLUDE_DIRS ""
+)
+# show all logs from this component, otherwise overridden by CONFIG_LOG_DEFAULT_LEVEL_ERROR
+target_compile_options(${COMPONENT_LIB} PRIVATE "-D LOG_LOCAL_LEVEL=ESP_LOG_INFO")
```

Comment:
Just a very minor thing - I'm inclined to leave all the config as basic as possible so unless there's a good reason for it I'd opt for a minimal CMakeLists.txt as with other examples